### PR TITLE
[ROCKETMQ-353] Add SendMessage Command

### PR DESCRIPTION
**Pull request by lindzh**
## What is the purpose of the change

In some cases , we want to send message menually to a specified topic in product environment , such as DLQ resend, we have to write code and publish. In my opinion , sendMessage command is commonly used in such cases.

## Brief changelog

Add SendMessageCommand

## Verifying this change

This is a trivial change.

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/ROCKETMQ/issues/) filed for the change (usually before you start working on it). Trivial changes like typos do not require a JIRA issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ROCKETMQ-XXX] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [x] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        return "Send Message to a topic";
+    }
+
+    @Override
+    public Options buildCommandlineOptions(Options options) {
+        Option opt = new Option("t", "topic", true, "topic name");
+        opt.setRequired(true);
+        options.addOption(opt);
+
+        opt = new Option("b", "body", true, "message body string utf-8 format");
```
Comment (vongosling) [inline tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java]: utf-8 string format of the message body

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+        opt = new Option("k", "key", true, "message keys");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("g", "tags", true, "message tags");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("q", "qbroker", true, "send message to which broker");
```
Comment (vongosling) [inline tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java]: q is not  a good abbreviation for broker

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java (hunk 1 above)
Comment (lindzh) [reply tools/src/main/java/org/apache/rocketmq/tools/command/message/SendMessageCommand.java]: Thx

**Comment by jisumanbu** [comment]
Glad to see this new feature.

**Comment by what-a-good-jungle** [comment]
I have made a new pull request to add SendMessageCommand and ConsumeMessageCommand base on this PR

**Comment by vongosling** [comment]
thanks @lindzh @what-a-good-jungle I will help you to polish the PR :-)

**Pull request closed without merging**